contrib/gin-gonic/gin/gintrace_test.go

```diff
@@ -167,3 +167,27 @@ func TestGetSpanNotInstrumented(t *testing.T) {
 	response := w.Result()
 	assert.Equal(response.StatusCode, 200)
 }
+
+func TestExtractSpan(t *testing.T) {
```

Comment:
Can we please call this `TestPropagation`? "Propagation" is the terminology we use throughout and will make it more obvious.